net/geoipupdate/Makefile

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=geoipupdate
+PKG_VERSION:=4.8.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/maxmind/geoipupdate/archive/refs/tags/
+PKG_HASH:=ca718c3ffcc595ef441363699888d20150f1d3a6583ac2d60bcbd34f052db09f
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_LICENSE:=Apache-2.0, MIT
+PKG_LICENSE_FILES:=LICENSE-APACHE, LICENSE-MIT
+PKG_MAINTAINER:=Matthew Hagan <mnhagan88@gmail.com>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/maxmind/geoipupdate
+GO_PKG_LDFLAGS_X:= \
+	main.defaultConfigFile=/etc/GeoIP.conf \
+	main.defaultDatabaseDirectory=/tmp/GeoIP
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/geoipupdate/Default
+  TITLE:=GeoIP Update
+  URL:=https://github.com/maxmind/geoipupdate
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/geoipupdate
+$(call Package/geoipupdate/Default)
+  SECTION:=net
+  CATEGORY:=Network
+endef
```

Comment:
You can merge this, right? You don't need to use `Package/geoipupdate/Default`. It is ok to use just this: `define Package/geoipupdate`

Reply:
Thanks. I did likewise for the description